README.md

```diff
@@ -2,7 +2,8 @@
 #This is MyPortfolio Project
     This is simple Project.Which has header, Section.Inside header there is logo and menu icon.
     Inside section some pictures. My itroduction in paragraph.I add a work section where details of my work. And after my work is my skill section where I add detailas of languages, frameworks and skills.And at the bottom there is footer, and inside footer I added a contact form.
-    In the stylesheet add media quries to make the portfolio desktop responsive.
+    In the stylesheet add media quries to make the portfolio desktop responsive. Also add github Pages to make my Portfolio online available. Here is the link :
+    https://ridaarif98.github.io/NewPortfolio/
 
   ![screenshot](./Images/newPort.png)
```

Comment:
- [ ] Good job here! Your readme is very descriptive but it needs a few tweaks and you're good to go!
- Although your readme is descriptive, it is not very organized. I understand adding images to your readme can help with the information, however, you do not need to add screenshots from all parts of the page. A single picture showing the mobile screen size is enough. You can also add a Gif showing the project. 
- The description as well needs some work, instead of saying:
> This is simple Project.Which has header, Section.Inside header there is logo and menu icon.    Inside section some pictures. My itroduction in paragraph.I add a work section where details of my work. And after my work is my skill section where I add detailas of languages, frameworks and skills.And at the bottom there is footer, and inside footer I added a contact form.

You can say:

> This is a mobile-first portfolio project built using Semantic HTML and CSS. It includes the navigation bar, the headline, a list of past projects(which includes skills, languages and frameworks used), the about section and a contact form.

You can see that my suggestion includes all of the things you mentioned above, but only this time it is more organized and articulated without sounding repetitive. 

- [ ] Include information on how to contact the author i.e a link to your social media
- [ ] Include acknowledgements to Figma for the design template
- [ ] Update the Prerequisites section to include necessary information on tools needed to run your project locally
- [ ] The information you currently have in the `Prerequisites` section should be under `Local Usage` which should include detailed information on how to clone and run the project locally.
Refer to this template [here](https://github.com/microverseinc/readme-template) and customise it to your needs.